From d7645b67c350f7179a1eba749ec4524c74948d86 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Thu, 29 Oct 2020 22:51:57 +0000
Subject: [PATCH] aclocal.m4: Fix compatibility with upcoming autoconf-2.70.

Under autoconf 2.69c (beta) `./configure` fails as:

```
./configure
...
checking for msgfmt... msgfmt
./configure: line 7441: syntax error near unexpected token `newline'
./configure: line 7441: `    '''
```

This happens due to underquoting of a snippet
argument for `AS_VAR_SET_IF` call.

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
Signed-off-by: Alexandre Julliard <julliard@winehq.org>
---
 aclocal.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aclocal.m4 b/aclocal.m4
index c5edd85cee49..bd8ca6b33e55 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -28,7 +28,7 @@ dnl Like AC_CHECK_TOOL but without the broken fallback to non-prefixed name
 dnl
 AC_DEFUN([WINE_CHECK_HOST_TOOL],
 [AS_VAR_SET_IF([ac_tool_prefix],
-  AC_CHECK_PROG([$1],[${ac_tool_prefix}$2],[${ac_tool_prefix}$2],,[$4]))
+  [AC_CHECK_PROG([$1],[${ac_tool_prefix}$2],[${ac_tool_prefix}$2],,[$4])])
 AS_VAR_IF([ac_cv_prog_$1],[],
   [AS_VAR_IF([cross_compiling],[yes],[],
     [AS_UNSET([ac_cv_prog_$1])
